src/rabbit.app.src

```diff
@@ -1,7 +1,7 @@
 {application, rabbit,           %% -*- erlang -*-
  [{description, "RabbitMQ"},
   {id, "RabbitMQ"},
-  {vsn, "0.0.0"},
+  {vsn, "3.7.0"},
```

Comment:
This value is updated at build time. @dumbbell do you agree with this default value bump?